mozilla_bitbar_devicepool/configuration.py

```diff
@@ -65,7 +65,7 @@ def configure(bitbar_configpath, filespath=None):
     FILESPATH=filespath
 
     with open(bitbar_configpath) as bitbar_configfile:
-        CONFIG = yaml.load(bitbar_configfile.read())
+        CONFIG = yaml.load(bitbar_configfile.read(), Loader=yaml.SafeLoader)
```

Comment:
Are we ok with the SafeLoader? We aren't loading untrusted content. Are we losing anything by not going with FullLoader ?

Reply:
device_group_report is already using safe_load (https://github.com/bclary/mozilla-bitbar-devicepool/blob/master/mozilla_bitbar_devicepool/device_group_report.py#L30). 

The SafeLoader seems to restrict data structures to simple ones vs full python objects. We can always go to FullLoader if/when we need to load python objects. Putting complex python objects into a configuration file kind of gives me a bad code smell.

from https://pyyaml.org/wiki/PyYAMLDocumentation#loading-yaml

```
Note that the ability to construct an arbitrary Python object may be dangerous if you receive a YAML document from an untrusted source such as the Internet. The function yaml.safe_load limits this ability to simple Python objects like integers or lists.

A python object can be marked as safe and thus be recognized by yaml.safe_load. To do this, derive it from yaml.YAMLObject (as explained in section Constructors, representers, resolvers) and explicitly set its class property yaml_loader to yaml.SafeLoader.
```